File: API/Entities/BingoCell.cs
using System;

namespace API.Entities;

public class BingoCell
{
    public int Id { get; set; }
    public float Score { get; set; }

    public int Row { get; set; }
    public int Column { get; set; }

    public Team Team { get; set; } = null!;
    public int OfficialScore { get; set; } = 0;

    public bool IsWrong { get; set; } = false;
}

File: client/src/app/_models/bingoCell.ts
import { Team } from "./team";


export interface BingoCell {
  id: number;
  score : number;
  createdAt: Date;
  updatedAt: Date;
  bingoId: number;
  row: number;
  column: number;
  team : Team;
  officialScore: number;
  isChecked: boolean;
}
